pkg/models/monitoring/expressions/prometheus/prometheus.go

```diff
@@ -0,0 +1,99 @@
+package prometheus
```

Comment:
this file is already in a folder called prometheus, is it better to change the file name prometheus.go to a more meaningful name like label_replace.go or others ?

Reply:
yep, label_replace.go looks meaningful. done